src/domain/validation-constants.js

```diff
@@ -13,6 +13,8 @@ export const minUserPasswordLength = 8;
 
 export const maxMediaLibraryItemDescriptionLength = 1000;
 
+export const maxCatalogPluginItemLinkDescriptionLength = 200;
```

Comment:
As discussed we could lift that restriction.

Reply:
🏋️‍♀️ lifted